website/src/content/docs/ja/reference/vscode.mdx

```diff
@@ -0,0 +1,141 @@
+---
+title: VSCode拡張機能
+description: BiomeのVSCode拡張機能について
+---
+# Biome VS Code 拡張機能
+
+[Biome](https://biomejs.dev/) は、開発スタックを統合し、複数のツールの機能を一つにまとめます。単一の設定ファイルを使用し、素晴らしいパフォーマンスで、あらゆるスタックで動作します。この拡張機能はBiomeをエディターで使えるようにし、以下のことができるようになります：
+
+- ファイルを*保存時*にフォーマット、*ドキュメントのフォーマット*コマンドの使用
+- タイプ中に静的解析の結果を確認し、コード修正を適用
+- リファクタリングの実行
+
+## インストール方法
+
+VS Codeの拡張機能をインストールするには、[Visual Studio Code マーケットプレイスページ](https://marketplace.visualstudio.com/items?itemName=biomejs.biome)にアクセスするか、VS Code内で以下の方法でインストールできます：
+
+- *拡張機能* タブ（_表示_ → _拡張機能_）を開き、Biomeを検索する。
+- _クイックオープンオーバーレイ_（<kbd>Ctrl</kbd>/<kbd title="Cmd">⌘</kbd>+<kbd>P</kbd>または _移動_ → _ファイルに移動_）で `ext install biomejs.biome` と入力し、Enterキーを押す。
```

Comment:
I think Command Palette is official word: https://code.visualstudio.com/docs/getstarted/userinterface#_command-palette
```suggestion
- _Command Palette_（<kbd>Ctrl</kbd>/<kbd title="Cmd">⌘</kbd>+<kbd>P</kbd>または _移動_ → _ファイルに移動_）で `ext install biomejs.biome` と入力し、Enterキーを押す。
```